TrecQA/build_qrels.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+for set in ['train-all', 'raw-test', 'raw-dev']:        
+    
+    with open(set + '.qrel', 'w') as qrelf:
+        labels = [y.strip() for y in open(os.path.join(set+'/sim.txt')).readlines()]
+        qids = [id.strip() for id in open(os.path.join(set+'/id.txt')).readlines()]
+
+        for i in range(len(labels)):
+            qid = qids[i]
+            lbl = labels[i]
+            ext_feats = None
```

Comment:
Is ext_feats needed here? if it's a no-op delete it.
I like the more pythonic way of zipping: https://docs.python.org/3/library/functions.html#zip See if you want to modify the loop to zip the files instead? 
You can refer: http://stackoverflow.com/questions/2323394/zip-alternative-for-iterating-through-two-iterables